app/helpers/publishers_helper.rb

```diff
@@ -63,6 +63,30 @@ def publisher_converted_overall_balance(publisher)
     end
   end
 
+  def publisher_referral_bat_balance(publisher)
```

Comment:
This might be better served in the `admin_helper`, or another helper file because these actions aren't displayed on the publishers homepage. Helpers are usually in the context of the route

EDIT: Okay I see, this is actually what's displayed on the admin, but we took the code from elsewhere?